operations/mimir-mixin/alerts/alerts-utils.libsonnet

```diff
@@ -11,4 +11,23 @@
 
   jobMatcher(job)::
     'job=~".*/%s"' % job,
+
+  withRunbookURL(url_format, groups)::
```

Comment:
We're already using `mixin-utils` lib, so we can just reuse the function defined there. You can import the utils using `local utils = import 'mixin-utils/utils.libsonnet';`

Reply:
The function in mixin-utils does not lower case the alert name. we need to lowercase the rule.alert for mimir alerts.

Reply:
I'm not sure case sensitivity is a real issue. We're already linking runbooks in Grafana Labs and we haven't noticed any issue.

Reply:
You're right. The reason why it works at Grafana Labs is because we link to GitHub page instead where the anchor keeps the case of the alert name. I think we could contribute upstream to pass an optional function to manipulate the link anchor, but let's keep it simple for now and merge it as is.

Thanks!

Reply:
Thank you.